smarts/sstudio/genhistories.py

```diff
@@ -117,6 +128,14 @@ def _create_tables(self, dbconxn):
                    FOREIGN KEY (vehicle_id) REFERENCES Vehicles(id)
                ) WITHOUT ROWID"""
         )
+        ccur.execute(
+            """CREATE TABLE Traffic_Lights (
```

Comment:
The convention we've been using is to have singular table names.  And I think it's better to treat table names like class names.  So I suggest this table be named `TrafficLight`.